Source/WebCore/rendering/RenderBlock.cpp

```diff
@@ -94,8 +96,8 @@ struct SameSizeAsRenderBlock : public RenderBox {
 
 static_assert(sizeof(RenderBlock) == sizeof(SameSizeAsRenderBlock), "RenderBlock should stay small");
 
-typedef HashMap<const RenderBlock*, std::unique_ptr<TrackedRendererListHashSet>> TrackedDescendantsMap;
-typedef HashMap<const RenderBox*, HashSet<const RenderBlock*>> TrackedContainerMap;
+using TrackedDescendantsMap = WeakHashMap<const RenderBlock, std::unique_ptr<TrackedRendererListHashSet>>;
```

Comment:
I believe that `TrackrdRendererListHashSet` is a single possibly null pointer to a hash table, and it is wasteful to store it in turn in a `unique_ptr`. We should consider fixing this mistake at some point.

Reply:
It is a pointer to the HashTable and 2 unsigned data members.